### PR TITLE
Minor changes for frontend fixes

### PR DESCRIPTION
**Pull request by angeljruiz**
Fixes a dashboard & events page error

### CODE REVIEW

**Comment by andrewc910** [comment]
Can you breakdown for me why each file required these changes? I can't figure it out.

File: app/models/bridge.rb [hunk 1]
```diff
@@ -51,12 +51,12 @@ class Bridge < ApplicationRecord
...
   def add_event_info
-    id, completed_at = latest_event_data
-
+    id, completed_at, created_at = latest_event_data
     attributes.merge!({
                         'eventCount' => events.count,
                         'completedAt' => completed_at || '',
-                        'eventId' => id
+                        'eventId' => id,
+                        'latestRequest' => created_at
```
Comment (andrewc910) [inline app/models/bridge.rb:59]: This is suppose to be for the latest inbound payload drop down right? If so, this should be `data['inbound']` not created_at.

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ def show
         event: @event,
         bridge_title: @event.bridge.title,
         bridge_slug: @event.bridge.slug,
-        events: fetch_events.to_json(only: %i[completed completed_at id status_code])
+        events: fetch_events.to_json(only: %i[aborted completed completed_at id status_code])
```
Comment (andrewc910) [inline app/controllers/events_controller.rb:21]: I believe this data is for the sidebar? Are we planning on changing the sidebar to say aborted or something? Can't see why we need this.

File: app/models/bridge.rb [hunk 3]
```diff
@@ -91,7 +91,7 @@ def latest_event_data
     events.where(completed: true)
           .order(completed_at: :desc)
           .limit(1)
-          .pluck(:id, :completed_at)
+          .pluck(:id, :completed_at, :created_at)
```
Comment (andrewc910) [inline app/models/bridge.rb:94]: This goes for the comment above. I think we need to pluck data and in the method above, pull inbound out.

File: app/models/bridge.rb (hunk 1 above)
Comment (andrewc910) [reply app/models/bridge.rb:59]: Also, mind adding a spec to ensure the controller returns the proper data? I think we are only testing if the controller works and not if the data returned is what is expected.

**Review by AndiLavera — CHANGES_REQUESTED** [review]

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (angeljruiz) [reply app/controllers/events_controller.rb:21]: Nah, this is necessary for the events page. 🙂 When an event is aborted the snackbar at the top will say aborted.

**Pull request merged**